doc/users/screenshots.rst

```diff
@@ -71,7 +71,7 @@ a vector field. In addition to simply plotting the streamlines, it allows you
 to map the colors and/or line widths of streamlines to a separate parameter,
 such as the speed or local intensity of the vector field.
 
-.. plot:: mpl_examples/images_contours_and_fields/streamplot_demo_features.py
+.. plot:: mpl_examples/images_contours_and_fields/streamplot_demo.py
```

Comment:
Here too, 'streamplot_demo.py' should be changed into 'plot_streamplot.py', shouldn't it?